static/docs/commands-reference/import-url.md

```diff
@@ -87,7 +93,7 @@ from having to install CLI tools for each service.
 
 When DVC inspects a DVC-file, its dependencies will be checked to see if any
 have changed. A changed dependency will appear in the `dvc status` report,
-indicating the need to reproduce this import stage. When DVC inspects an
```

Comment:
"import stage" is kind of a noun phrase . It comes from code again. I'm not sure if there is a better term. But imported is not the right word for sure.

Reply:
You're right. "import stage" is fine. Reverted in 2915e33